### ISSUE TITLE
Mehrdad Ghyabi

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#168]
Phd (Engineering) + Masters (Data Science) George Mason University

**Comment by aflies1** [comment]
Availability - has the availability and understands the time commitment and how we work daily. Has his green card, so can sign on right away (I believe he is from Iran, but with a green card not sure if that's an issue.) 

Python - Seems well versed in Python. Has coded since he was a kid and has used primarily Pythin in all his academic work over the last few years.

SQL- Familiar with this Data base, but not sure how much he has used it.

ML- Seems like all his experience in ML has been academic, Louis will need to dive much deeper into his knowledge.

**Comment by ldaudet** [comment]
Mehrdad Ghyabi

He got a master degree in engineering, after a BA in engineering, and he is now doing a MS in data analytics and a phd in engineering at eh same time as a bundle deal.

The data analytics engineering is a misnomer of sorts. The engineering part comes from being taught in the engineering school, so it's really a data analytics. Haven't used XGboost surprisingly, but should be able to use it with no issue as he know ADABoost, that he has coded from scratchin one of their project, so he should be able to get on top of it easy.

The signal fusion using Gans is an interesting project, which we discussed for a few minutes, but has very little to do with the work at USAA, besides the fact that it shows proficiency in NN.

He didn't list it on his resume but he work in Iran as an ingeneer, so he is familiar with the work place. 

He decided to stop working as an engineer, becasue he always wanted to get a PhD at some point. 

I described the problem we are tackling now and his approach was spot on. 

Sql is intermediate. NLP and hypothesis testing experience.

Intermediate in github, linux, and shell.

He had a side job for 20 hrs last semester and was fine, so 16 hours should not be a worry.

TL;DR: He would be a GREAT fit I think
